.gitignore

```diff
@@ -428,3 +428,6 @@ website/themes
 website/sphinx/_build
 website/sphinx/api
 website/python_api
+
+# python code coverage
+coverage
```

Comment:
does that need to go in the top level?

Reply:
Actually, seems unrelated for this PR, no?

Reply:
depends? Not necessarily, that's the default if this is where we run the python tests from.

Reply:
k, cool